management-service/controller/experiment_controller_test.go

```diff
@@ -81,7 +81,8 @@ func (s *ExperimentControllerTestSuite) SetupSuite() {
 			"type": "",
 			"start_time": "0001-01-01T00:00:00Z",
 			"updated_at": "0001-01-01T00:00:00Z",
-			"updated_by": ""
+			"updated_by": "",
+			"version": 0
```

Comment:
nit: Since we expect versions to start from 1, shall we use 1 instead for these tests?

Reply:
I intentionally left that field (and previously other properties too) empty - they don't play a role in the test cases. So they are empty values by default.